modules/virt-restoring-vm-from-snapshot-cli.adoc

```diff
@@ -12,6 +12,12 @@ You can restore an existing virtual machine (VM) to a previous configuration by
 
 * Power down the VM you want to restore.
 
+* Optional: Adjust what happens if the target VM is not ready. To do so, set the `targetReadinessPolicy` parameter in the `vmrestore` YAML configuration to one of the following values:
+** `FailImmediate`` - The restoration process fails immediately if the VM is not ready. This is the default setting.
+** `StopTarget` - If not ready, the VM is stopped.
+** `WaitGracePeriod _<10>_` - The restoration process waits for a set amount of time, in minutes, for the VM to be ready. The default value is 5 minutes.
```

Comment:
what is <10> stands for, if its the default time it should be 5 as mentioned

Reply:
I wanted to show a non-default value, but if this parameter is set and active by default, you're right that 5 would be better here.